freshmaker/lightblue.py

```diff
@@ -712,18 +712,19 @@ def _get_default_projection(self, srpm_names=None, include_rpm_manifest=True):
             {"field": "architecture", "include": True, "recursive": False},
         ]
         if include_rpm_manifest:
-            if srpm_names:
+            if rpm_names:
                 projection += [
                     {"field": "rpm_manifest.*.rpms", "include": True, "recursive": True,
                      "match": {
                          "$or": [{
-                             "field": "srpm_name",
+                             "field": "name",
                              "op": "=",
-                             "rvalue": srpm_name
-                         } for srpm_name in srpm_names]},
+                             "rvalue": rpm_name
+                         } for rpm_name in rpm_names]},
                      "project": [
                          {"field": "srpm_nevra", "include": True},
```

Comment:
Not sure if we still need 'srpm_nevra' and 'srpm_name' fields

Reply:
We can keep them at this moment, you can test and remove them in other PR, remember to update the odcs compose releated  code when you're going to remove `srpm_nevra`.

Reply:
Sure, thx